src/hdr_histogram.h

```diff
@@ -33,8 +33,36 @@
     using std::memory_order_seq_cst;
 
     using std::atomic_int_least64_t;
-#else /* for std C11 compiler */
-    #include <stdatomic.h>
+#else /* for std C compiler */
+    #ifdef _MSC_VER /* for MSVC as it dosn't support stdatomic.h */
+        /*
+         * MSVC dose not support stdatomic.h  so we have to use a volatile
+         * variable and Interlock* methods
+         *
+         * MSVC treats volatile variables difrently by defualt and automaticly
+         * provids higher memory ordering than an ISO compliant volatile.
+         *
+         * See link bellow for more information.
+         * https://docs.microsoft.com/en-us/cpp/cpp/volatile-cpp?view=vs-2017
+         */
+        #define WIN32_LEAN_AND_MEAN
+        #include <Windows.h>
+
+        typedef volatile int64_t atomic_int_least64_t;
+        #define memory_order_relaxed 0
```

Comment:
Do you even need this now? Given the memory orderings are never used in any of your macros below.

Reply:
I define this as a macros as otherwise the code in hdr_histogram.c wouldn't be able to find anything called "memory_order_relaxed" as its required for stdatomic.h. Though I probably don't need to define it too anything just that it's there.